src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -38,14 +38,29 @@ class PostgreSQLSchemaManager extends AbstractSchemaManager
     /**
      * Gets all the existing schema names.
      *
+     * @deprecated Use {@link listSchemaNames()} instead.
+     *
      * @return string[]
      *
      * @throws Exception
      */
     public function getSchemaNames()
+    {
+        return $this->listNamespaceNames();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function listSchemaNames(): array
     {
         return $this->_conn->fetchFirstColumn(
-            "SELECT nspname FROM pg_namespace WHERE nspname !~ '^pg_.*' AND nspname != 'information_schema'"
+            <<<'SQL'
+SELECT schema_name
+FROM   information_schema.schemata
+WHERE  schema_name NOT LIKE 'pg\_%'
+AND    schema_name != 'information_schema'
+SQL
```

Comment:
PHP 7.3 allows you to use indention for nowdoc here and below :slightly_smiling_face:

Reply:
I have mixed feelings about this feature:
1. The left margin of the block is not clearly visible.
2. It's usually a program within a program, so it's fine that it's not indented the same as the outer one.

So far, it's consistent with the rest of the codebase (except for https://github.com/doctrine/dbal/pull/4483). Let's reconsider this at the coding standard level first.